docs/CustomResources.md

```diff
@@ -4,14 +4,15 @@ The Splunk Operator provides a collection of
 [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
 you can use to manage Splunk Enterprise deployments in your Kubernetes cluster.
 
-* [Metadata Parameters](#metadata-parameters)
-* [Common Spec Parameters for All Resources](#common-spec-parameters-for-all-resources)
-* [Common Spec Parameters for All Splunk Enterprise Resources](#common-spec-parameters-for-all-splunk-enterprise-resources)
-* [Spark Resource Spec Parameters](#spark-resource-spec-parameters)
-* [LicenseMaster Resource Spec Parameters](#licensemaster-resource-spec-parameters)
-* [Standalone Resource Spec Parameters](#standalone-resource-spec-parameters)
-* [SearchHeadCluster Resource Spec Parameters](#searchheadcluster-resource-spec-parameters)
-* [IndexerCluster Resource Spec Parameters](#indexercluster-resource-spec-parameters)
+- [Custom Resource Guide](#custom-resource-guide)
+  - [Metadata Parameters](#metadata-parameters)
+  - [Common Spec Parameters for All Resources](#common-spec-parameters-for-all-resources)
+  - [Common Spec Parameters for Splunk Enterprise Resources](#common-spec-parameters-for-splunk-enterprise-resources)
+  - [Spark Resource Spec Parameters](#spark-resource-spec-parameters)
+  - [LicenseMaster Resource Spec Parameters](#licensemaster-resource-spec-parameters)
+  - [Standalone Resource Spec Parameters](#standalone-resource-spec-parameters)
+  - [SearchHeadCluster Resource Spec Parameters](#searchheadcluster-resource-spec-parameters)
```

Comment:
Shouldn't we add a note for the CM CR too?

Reply:
Hmm I think we can. ClusterMaster CR does not have any required spec as such other than CommonSplunkSpec, but we can give an example of S2 configuration settings in the CM CR.

Reply:
Done